api/clients/v2/config.go

```diff
@@ -194,7 +194,7 @@ func GetDefaultValidatorPayloadRetrieverConfig() *ValidatorPayloadRetrieverConfi
 	return &ValidatorPayloadRetrieverConfig{
 		PayloadClientConfig: *getDefaultPayloadClientConfig(),
 		RetrievalTimeout:    20 * time.Second,
-		ConnectionCount:     10,
+		MaxConnectionCount:  10,
```

Comment:
shouldn't this be like 100? With 200 validators, assuming each call takes 1 sec, just hitting all the validators already takes 20 seconds.

Reply:
Good point. I increased the connection count to `100`, and also increased retrieval timeout to 30 seconds, to add a little buffer.

```
		RetrievalTimeout:    30 * time.Second,
		MaxConnectionCount:  100,
```